src/IssueViz.Security/HotspotsList/HotspotsControl.xaml.cs

```diff
@@ -54,8 +62,30 @@ private static IWpfTableControl CreateWpfTableControl(ITableManagerProvider tabl
             return wpfTableControl;
         }
 
+        private async void SelectionService_SelectionChanged(object sender, SelectionService.SelectionChangedEventArgs e)
+        {
+            if (e.SelectedHotspot == null)
+            {
+                return;
+            }
+
+            await wpfTableControl.ForceUpdateAsync();
```

Comment:
This needs a comment to explain why it's necessary (particularly since the docs say _"There should be no need to call this method (updates will happen automatically if there is a change). It is exposed here primarily to simplify testing."_)